ckanext/datarequests/fanstatic/webassets.yml

```diff
@@ -0,0 +1,15 @@
+edit_comments-js:
```

Comment:
how does this work, link to doc's for others to learn also please

Reply:
Added a comment linking to https://docs.ckan.org/en/2.9/contributing/frontend/assets.html